index.html

```diff
@@ -0,0 +1,2620 @@
+                    production has been reported to be...
+                  </p>
+                </div>
+              </div>
+
+              <div class="media border-top border-bottom py-3">
+                <img
+                  src="img/cobra-kai-karate-kid-sequel.webp"
+                  class="float-left mr-3"
+                  alt="froma"
+                />
+                <div class="article-body">
+                  <p class="card-title pt-1 mb-1 font-weight-bold">
+                    'Cobra Kai' Season 4 Expected to Start Shooting Today
+                  </p>
+                  <p class="card-text text-secondary">
+                    "Cobra Kai" Season 4 has been confirmed by Netflix, and
+                    production has been reported to be...
+                  </p>
+                </div>
+              </div>
+
+              <div class="media border-top border-bottom py-3">
+                <img
+                  src="img/cobra-kai-karate-kid-sequel.webp"
+                  class="float-left mr-3"
+                  alt="froma"
+                />
+                <div class="article-body">
+                  <p class="card-title pt-1 mb-1 font-weight-bold">
+                    'Cobra Kai' Season 4 Expected to Start Shooting Today
+                  </p>
+                  <p class="card-text text-secondary">
+                    "Cobra Kai" Season 4 has been confirmed by Netflix, and
+                    production has been reported to be...
+                  </p>
+                </div>
+              </div>
+
+              <div class="media border-top py-3">
+                <img
+                  src="img/cobra-kai-karate-kid-sequel.webp"
+                  class="float-left mr-3"
+                  alt="froma"
+                />
+                <div class="article-body">
+                  <p class="card-title pt-1 mb-1 font-weight-bold">
+                    'Cobra Kai' Season 4 Expected to Start Shooting Today
+                  </p>
+                  <p class="card-text text-secondary">
+                    "Cobra Kai" Season 4 has been confirmed by Netflix, and
+                    production has been reported to be...
+                  </p>
+                </div>
+              </div>
+
+              <!-- MY TURN -->
+              <div class="col-12 p-1">
+                <h6 class="text-secondary">MY TURN</h6>
+                <div class="row">
+                  <div class="card border-0 col-4">
+                    <img
+                      src="img/marriage-same-sex-marriage-best-friends.jpg"
+                      class="card-img-top"
+                      alt="tax"
+                    />
+                    <div class="article-body">
+                      <p class="card-text text-secondary">
+                        'I Married My Best Friend to Avoid Inheritance Tax'
+                      </p>
+                    </div>
+                  </div>
```

Comment:
- [ ] The "My Turn" and "Culture and travel" sections of your project are not responsive at all.
Please make sure they behave responsively, as they do in the original page, in medium and small screens.

Reply:
done